shopify_app.gemspec

```diff
@@ -8,9 +8,7 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6', '< 5.0')
```

Comment:
I don't see this particular post relevant in our case, as it looks like Yehuda writes in the context of *duplication* in single gemspec and single Gemfile, which is obviously is not the case.
Ironically - duplication is the very thing I intended to avoid by explicitly putting differentiating versions of dependency in different Gemspecs.
With that being said, perhaps idea of not using Gemspec files for different purposes looks appealing:
```Gemspec
Gemspec.ci.rails-master
Gemspec.ci.rails-5-stable
Gemspec.ci.rails-4.2-stable